src/Service.Tests/SqlTests/RestApiTests/Find/FindApiTestBase.cs

```diff
@@ -1015,7 +1014,7 @@ public async Task FindTestWithFirstTwoVerifyAfterFormedCorrectlyWithOrderBy()
             string after = $"[{{\"EntityName\":\"Author\",\"FieldName\":\"birthdate\",\"FieldValue\":\"2001-01-01\",\"Direction\":0}}," +
                            $"{{\"EntityName\":\"Author\",\"FieldName\":\"name\",\"FieldValue\":\"Aniruddh\",\"Direction\":0}}," +
                            $"{{\"EntityName\":\"Author\",\"FieldName\":\"id\",\"FieldValue\":125,\"Direction\":1}}]";
-            after = $"&$after={Uri.EscapeDataString(SqlPaginationUtil.Base64Encode(after))}";
+            after = $"&$after={SqlPaginationUtil.Base64Encode(after)}";
```

Comment:
For consistency, should other tests which have long `after` values get base64 encoded like this prior to being passed in `expectedAfterQueryString`

Reply:
ill look through the code and try to get it consistent, this is kind of an on-going theme where we have some inconsistencies in style.

Reply:
since the test functionality isnt affected by this, I'm closing this thread. thanks for the background.